config.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"encoding/json"
+	xds "github.com/cncf/xds/go/xds/type/v3"
```

Comment:
Should sort it with goimports

Reply:
Resolved.

Reply:
@yizhibian 
No, it didn't.

And please do not mark a conversation as resolved without the confirm from the reviewer.

You can apply a patch like this:
```
diff --git a/config.go b/config.go
index 59e2a4b..766f255 100644
--- a/config.go
+++ b/config.go
@@ -2,11 +2,12 @@ package main
 
 import (
        "encoding/json"
+       "time"
+
        xds "github.com/cncf/xds/go/xds/type/v3"
        "github.com/envoyproxy/envoy/contrib/golang/filters/http/source/go/pkg/api"
        "github.com/envoyproxy/envoy/contrib/golang/filters/http/source/go/pkg/http"
        "google.golang.org/protobuf/types/known/anypb"
-       "time"
 )
 
 var Start time.Time
```

Reply:
Sorry, Resolved now.